packages/form-js-viewer/src/render/components/Sanitizer.js

```diff
@@ -7,6 +7,7 @@ const ALLOWED_NODES = [
   'h3',
   'h4',
   'h5',
+  'h6',
```

Comment:
Maybe let's update our [existing test case](https://github.com/bpmn-io/form-js/blob/develop/packages/form-js-viewer/test/spec/render/components/form-fields/Text.spec.js#L41) to also test for the new `h6` element.